app/src/main/res/values-sv/strings.xml

```diff
@@ -142,6 +154,12 @@
         Ingen kompatibel epost app är installerad.
     </string>
     <string name="notifications_schedule_update_channel_name">Schemalägg uppdateringar</string>
+    <plurals name="notifications_schedule_changes">
+        <item quantity="one"><xliff:g example="1" id="count">%d</xliff:g> föreläsning har ändrats.</item>
+        <item quantity="few"><xliff:g example="2" id="count">%d</xliff:g> föreläsningar har ändrats.</item>
+        <item quantity="many"><xliff:g example="3" id="count">%d</xliff:g> föreläsningar har ändrats.</item>
+        <item quantity="other"><xliff:g example="3" id="count">%d</xliff:g> föreläsningar har ändrats.</item>
```

Comment:
This can be simplified by just using "other". See https://developer.android.com/guide/topics/resources/string-resource#Plurals.